climada/hazard/test/test_tc_tracks_synth.py

```diff
@@ -356,6 +356,27 @@ def test_wrong_decay_pass(self):
         self.assertTrue(np.array_equal(cp_ref, track_res.central_pressure[9:11]))
 
```

Comment:
It would be useful to also have unit test for new methods _get_destinations_points, _get_angular_distance, _get_bearing_angle, _random_uniform_ac, _f_ac_, _h_ac, ...

Reply:
Actually there is in test_angle_funs_pass for the following:
- _get_bearing_angle
- _get_angular_distance
- _get_destination_points
But for the ones related to random uniform I must admit I won't find time...